ceph.spec.in

```diff
@@ -1266,6 +1317,12 @@ fi
 %{python_sitearch}/rados-*.egg-info
 
 #################################################################################
+%files -n python%{python3_pkgversion}-rados
+%defattr(-,root,root,-)
+%{python3_sitearch}/rados.cpython*.so
+%{python3_sitearch}/rados-*.egg-info
+
+#################################################################################
```

Comment:
I don't see how to do this consistently, because the new packages go in between the old packages.

I'm afraid these two PRs will conflict no matter what, so I think it makes more sense to keep things as is.